locales/en/kafkaoverview-v2.json

```diff
@@ -5,15 +5,13 @@
   "contactSalesCardMainText": "Contact sales to get a pre-paid subscription or modify limits of your current subscription.",
   "contactSalesCardTitle": "Contact sales",
   "dataTransfer": "Data transfer",
+  "dataTransfer_price": "$0.09 / GB",
   "firstStreamingUnit": "1 streaming unit",
-  "heroCallToActionButton": "",
   "heroDescription": "Red Hat OpenShift Streams for Apache Kafka is a managed cloud service that provides a streamlined developer experience for building, deploying, and scaling real-time applications in hybrid-cloud environments. The combination of seamless operations across distributed microservices, large data transfer volumes, and managed operations allows teams to focus on core competencies, accelerate application velocity and reduce operational cost.",
   "heroTagline": "Fully hosted and managed service for stream-based applications",
   "heroTitle": "Get started with Red Hat OpenShift Streams for Apache Kafka",
-  "heroTryItButton": "",
   "kafkaInstanceCapacityDescription": "OpenShift Streams offers instances in the following sizes with the following capacity.",
-  "kafkaInstanceCapacityfooter1": "*Storage can be extended. If you wish to increase the storage limit for a Kafka instance, contact",
```

Comment:
@HemaHG try not to split translation strings into substrings that are then concatenated back in the HTML. It makes reasoning about the copy more complicated than it should.